.circleci/config.yml

```diff
@@ -551,6 +551,24 @@ jobs:
           environment:
             VARIANT: '_partial'
             SUBDIR: 'fmriprep-partial'
+      - run:
+          name: Re-run fMRIPrep on single run of task data, without T1w in BIDS
```

Comment:
ds005 already takes extra time compared to the other tests. It would be nice if we could really bring this runtime down, or else move to a different dataset.

Reply:
Added test now run in 2m11s, is that fast enough or should I try with another dataset?